src/aks-preview/azext_aks_preview/_validators.py

```diff
@@ -219,3 +223,29 @@ def validate_user(namespace):
     if namespace.user.lower() != "clusteruser" and \
             namespace.user.lower() != "clustermonitoringuser":
         raise CLIError("--user can only be clusterUser or clusterMonitoringUser")
+
+
+def validate_load_balancer_outbound_ports(namespace):
```

Comment:
Add these rules to help. Otherwise, users need to retry many times to select a valid value.

Reply:
Added